phpcs.xml

```diff
@@ -1,9 +1,18 @@
 <?xml version="1.0"?>
-<ruleset name="Zend Framework coding standard">
-    <rule ref="./vendor/zendframework/zend-coding-standard/ruleset.xml"/>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+    <arg name="basepath" value="."/>
+    <arg name="cache" value=".phpcs-cache"/>
```

Comment:
Does this need to be added to gitignore?

Reply:
Not sure. Either remove it or add it to .gitignore.